result/metrics.yaml

```diff
@@ -14,72 +14,77 @@ components:
               x-field-uid: 1
             flow:
               x-field-uid: 2
-            bgpv4:
+            convergence:
               x-field-uid: 3
```

Comment:
This will mess everything up .. cannot change existing x-field-uids . Breaks old scripts and setups completely.

Reply:
Note : Before checkin , should ensure with debug setup it does not break existing tests since compilation of model does not catch this huge problem. Dependant solely on carefully doing this and testing.

Reply:
Ok. Append new one at the end to maintain existing x-field-uids.